siriuspy/siriuspy/injctrl/main.py

```diff
@@ -430,11 +430,11 @@ def set_multbunbiasvolt(self, value):
     def _set_egunbias(self, value):
         self.run_callbacks('BiasVoltCmdSts-Mon', _Const.IdleRunning.Running)
 
-        self._update_log('Setting EGun Bias voltage to {}V...'.format(value))
+        self._update_log(f'Setting EGun Bias voltage to {value:.3f}V...')
```

Comment:
I don't think we need to know the bias with this much precision. Maybe 2 decimal places is enough, what do you think @anacso17 ?

Reply:
done!